docs/docs/pixabay-source-plugin-tutorial.md

```diff
@@ -57,6 +57,8 @@ gatsby new source-tutorial-site https://github.com/gatsbyjs/gatsby-starter-defau
 cd source-tutorial-site
 ```
 
+> **NOTE:** This tutorial assumes you have [Gatsby CLI installed](https://www.gatsbyjs.org/docs/quick-start/). If you do not have the CLI installed, you can use `npx`, e.g., `npx gatsby new source-tutorial-site https://github.com/gatsbyjs/gatsby-starter-default`.
```

Comment:
We do not encourage npx usage, we push people to global install `gatsby-cli`. It might make more sense to tell people to visit https://www.gatsbyjs.org/docs/quick-start/ when they get "gatsby is not a operation" (I don't know the exact error message).

How does that sound? Thank you for making our docs better!

Reply:
@wardpeet I like that! Thanks for the suggestions! I'll update.